packages/cli-hydrogen/src/cli/services/deploy/config.ts

```diff
@@ -42,6 +42,7 @@ export const fillDeployConfig = async (config: DeployConfig): Promise<ReqDeployC
 
   return {
     ...config,
+    pathToBuild: config.pathToBuild ? path.resolve(config.pathToBuild) : '',
```

Comment:
Is `''` the actual value that we want or should it be `undefined`?

Reply:
It seems like every time I have an internal debate as to wether or not I should do something you have a question on it 🤣 , haha!

So we have two config types, `DeployConfig` & `ReqDeployConfig`. Both have the same properties but the second requiring all properties. First type is used for the CLI argument parsing and second type is used for `fillDeployConfig`. I agree that `undefined` would be the best solution but it just doesn't fit in our pattern/structure. I thought of using `string | null` instead of only `string` but both `''` & `null` evaluate to false so I chose to use empty string as the "no value" value.